eng/Microsoft.DotNet.Arcade.Sdk/tools/SourceBuildArcade.targets

```diff
@@ -0,0 +1,113 @@
+<Project>
+
+  <!--
+    These targets inject source-build into Arcade's build process.
+  -->
+
+  <PropertyGroup>
+    <SourceBuildEngineeringDir>$([MSBuild]::NormalizeDirectory('$(RepositoryEngineeringDir)', 'source-build'))</SourceBuildEngineeringDir>
```

Comment:
What is this property used for?  I don't see it anywhere else.

Reply:
This is a leftover line from my prototype that I missed deleting--the directory itself doesn't exist either. Good catch, removing.